modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -52,108 +54,107 @@ spec:
+      pi_integral_const                 0.0
+      pi_proportional_scale             0.0
+      pi_proportional_exponent         -0.3
+      pi_proportional_norm_max          0.7
+      pi_integral_scale                 0.0
+      pi_integral_exponent              0.4
+      pi_integral_norm_max              0.3
+      step_threshold                    0
+      first_step_threshold              0.00002
+      max_frequency                     900000000
+      clock_servo                       pi
+      sanity_freq_limit                 200000000
+      ntpshm_segment                    0
       #
       # Transport options
       #
-      transportSpecific   0x0
-      ptp_dst_mac          01:1B:19:00:00:00
-      p2p_dst_mac          01:80:C2:00:00:0E
-      udp_ttl                1
-      udp6_scope           0x0E
-      uds_address          /var/run/ptp4l
+      transportSpecific                 0x0
+      ptp_dst_mac                       01:1B:19:00:00:00
+      p2p_dst_mac                       01:80:C2:00:00:0E
+      udp_ttl                           1
+      udp6_scope                        0x0E
+      uds_address                       /var/run/ptp4l
       #
       # Default interface options
       #
-      clock_type             BC
-      network_transport    UDPv4
-      delay_mechanism        E2E
-      time_stamping          hardware
-      tsproc_mode            filter
-      delay_filter           moving_median
-      delay_filter_length  10
-      egressLatency          0
-      ingressLatency         0
-      boundary_clock_jbod  1
+      clock_type                        BC
+      network_transport                 L2
+      delay_mechanism                   E2E
+      time_stamping                     hardware
+      tsproc_mode                       filter
+      delay_filter                      moving_median
+      delay_filter_length               10
+      egressLatency                     0
+      ingressLatency                    0
+      boundary_clock_jbod               0           <9>
       #
       # Clock description
       #
-      productDescription    ;;
-      revisionData            ;;
-      manufacturerIdentity  00:00:00
-      userDescription         ;
-      timeSource              0xA0
-    phc2sysOpts: "-a -r" <9>
-    ptpSchedulingPolicy: SCHED_OTHER <10>
-    ptpSchedulingPriority: 65 <11>
-  ptpClockThreshold: <12>
-    holdOverTimeout: 5
-    maxOffsetThreshold: 100
-    minOffsetThreshold: -100
-  recommend: <13>
-  - profile: "profile1" <14>
-    priority: 10 <15>
-    match: <16>
-    - nodeLabel: "node-role.kubernetes.io/worker" <17>
-      nodeName: "compute-0.example.com" <18>
+      productDescription                ;;
+      revisionData                      ;;
+      manufacturerIdentity              00:00:00
+      userDescription                   ;
+      timeSource                        0xA0
+    phc2sysOpts:                        "-a -r"     <10>
```

Comment:
There is a mismatch between domainNumber value in the profile and this command line without **-n 24** value. The default domainNuber is zero. I've seen that you mention the right command line below, but leaving the profile with this mismatches might lead to confusion.